Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml

```
@@ -0,0 +1,381 @@
+    quietmode: 0
+    scriptarguments:
+      gridfield:
+        simple: asmserviceowner
+      keys:
+        simple: Name,Email,Source,Timestamp
+      val1:
+        simple: n/a
+      val2:
+        complex:
+          accessor: members
+          filters:
+          - - left:
+                iscontext: true
+                value:
+                  simple: GCPIAM.Policy.bindings.role
+              operator: isEqualString
+              right:
+                value:
+                  simple: roles/owner
+          root: GCPIAM.Policy.bindings
+          transformers:
+          - args:
+              limit: {}
+              replaceWith: {}
+              toReplace:
+                value:
+                  simple: 'user:'
+            operator: replace
+      val3:
+        simple: Owner of GCP project where service account is defined
+      val4:
+        complex:
+          root: TimeNowUnix
+          transformers:
+          - operator: TimeStampToDate
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Example of command:
+        `!GridFieldSetup keys=ip,src val1=${AWS.EC2.Instances.NetworkInterfaces.PrivateIpAddress} val2="AWS" gridfiled="gridfield"`
+      id: 00298ed8-f84b-4444-8207-50f3af9ec22d
+      iscommand: false
+      name: Add project owner to service owner grid field
+      script: GridFieldSetup
+      type: regular
+      version: -1
+    taskid: 00298ed8-f84b-4444-8207-50f3af9ec22d
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1190
+        }
+      }
+  "6":
+    id: "6"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    note: false
+    quietmode: 0
+    scriptarguments:
+      owners:
+        simple: ${alert.asmserviceowner}
+    separatecontext: false
+    skipunavailable: true
+    task:
+      brand: ""
+      description: Recommend most likely service owners from those surfaced by Cortex ASM Enrichment.
+      id: 7eb4fb8d-14f3-4c09-82ae-6b5893d7f8c4
+      iscommand: false
+      name: Normalize and rank likely service owners
+      script: RankServiceOwners
+      type: regular
+      version: -1
+    taskid: 7eb4fb8d-14f3-4c09-82ae-6b5893d7f8c4
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 200,
+          "y": 1420
+        }
+      }
+  "7":
+    id: "7"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "2"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      gridfield:
+        simple: asmserviceownerunrankedraw
+      keys:
+        simple: Name,Email,Source,Timestamp
```

```suggestion
        simple: name,email,source,timestamp
```

Please lowercase these, unless the RE / Platform team is using this today. I'll handle `asmserviceowner` on a later date.